fitlins/interfaces/bids.py

```diff
@@ -393,18 +393,18 @@ def _run_interface(self, runtime):
         mask_files = []
         entities = []
         for ents in self.inputs.entities:
-            selectors = {**self.inputs.selectors, **ents}
+            selectors = {**self.inputs.selectors, 'desc': 'preproc', **ents}
```

Comment:
By putting it first, if `'desc'` is in `selectors` or `ents`, it will override.

```suggestion
            selectors = {'desc': 'preproc', **self.inputs.selectors, **ents}
```

Unless your goal was specifically to override selectors and be overridden by ents?

Reply:
Nope, that works.